prime-router/docs/onboarding-users/how-to-filter-by-individual-LOINC-codes.md

```diff
@@ -0,0 +1,128 @@
+# [17171] How to Filter by Individual LOINC Codes
```

Comment:
Thinking out loud here, are there other docs where there is a natural spot to leave a breadcrumb trail to here? Maybe on one of the filtering/condition code docs, or the one for the receiver filter step?